### ISSUE TITLE
Error during converting a map

### ISSUE THREAD

**Issue by RealKalsep** [CloneTrooper1019/Source2Roblox#10]
The map that I'm converting is d1_trainstation_01, the same map as the example command.

Error happened during `Wrote C:\Users\(insert epic username here)\AppData\Local\Roblox Studio\content\source\hl2\models/props_combine/citadel_pods.rbxm`

```
System.NullReferenceException
  HResult=0x80004003
  StackTrace:
   at RobloxFiles.DataTypes.CFrame.op_Multiply(CFrame a, CFrame b) in D:\source2roblox\Roblox-File-Format\DataTypes\CFrame.cs:line 224
```
Line is `float[] ac = a.GetComponents();`

**Comment by shadow01148** [comment]
Mine worked fine? 
![image](https://user-images.githubusercontent.com/76457846/133530110-61e4326d-2dc0-43cd-b79a-b9778f487f50.png)

**Comment by CloneTrooper1019** [comment]
That's... strange, I haven't seen error triggered before.
Does it happen 100% of the time or with any other maps?

**Comment by RealKalsep** [comment]
Every time I tried to convert a map it did that. I use all the newest source code that was available at the time of making this thread from all the repositories.

**Comment by RealKalsep** [comment]
Just redownloaded the repositories + got the latest studio version from the mod manager and I'm still getting the error.

**Comment by shadow01148** [comment]
Have you tried rebuilding?

**Comment by RealKalsep** [comment]
Yes I have tried that yesterday and then running the program again. To no avail.

**Comment by shadow01148** [comment]
I honestly don't know what's the issue. Maybe try doing this?
![image](https://user-images.githubusercontent.com/76457846/133878350-c380b9bf-15da-4082-adf3-3a2028a3337a.png)

**Comment by RealKalsep** [comment]
Did that and I still get the "not found" error, after unpacking it still crashes.

**Comment by MaximumADHD** [comment]
It sounds like the code is trying to multiply a `CFrame` with `null`, or `null` with `CFrame`. The information you are providing here is not nearly enough for me to track down or reproduce how you are triggering this bug.

Are you running this in debug mode from Visual Studio 2019? If so, does it trigger a break at line 224 in `CFrame.cs`?
What does your callstack look like? Where is this multiplication even happening? I need more information or I can't help you.

**Comment by RealKalsep** [comment]
I'm running it in debug mode with VS2019, I tried putting it into release mode but that didn't fix it. And yes, the error is on line 224.
Callstack:
```
RobloxFileFormat.dll!RobloxFiles.DataTypes.CFrame.operator *(RobloxFiles.DataTypes.CFrame a, RobloxFiles.DataTypes.CFrame b) Line 224	C#
Source2Roblox.exe!Extensions.PivotTo(RobloxFiles.PVInstance pv, RobloxFiles.DataTypes.CFrame cf) Line 228	C#
Source2Roblox.exe!Source2Roblox.Geometry.MeshBuilder.BakeBSP_RBXL(Source2Roblox.World.BSPFile bsp, Source2Roblox.FileSystem.GameMount game) Line 1024	C#
Source2Roblox.exe!Source2Roblox.Program.Main(string[] args) Line 169	C#

```

**Comment by RealKalsep** [comment]
Has anyone figured it out? I have not touched the code since then.

**Comment by shadow01148** [comment]
Maybe reinstall Half-Life 2? I don't know lol

**Comment by RealKalsep** [comment]
I don't think it's because of the game, and verifying integrity would reinstall all the files that were modified or corrupted.

**Comment by shadow01148** [comment]
Ive never had this error before, so probably just delete everything and redo the steps listed in the README.md

**Comment by RealKalsep** [comment]
I did that 2 times, and it did not help. I don't know what I'm doing wrong then. I also tried changing the directory, nothing. Question, does this app work on other drives than C:?

**Comment by shadow01148** [comment]
Yes, it works with other drives

**Comment by RealKalsep** [comment]
Update, I redownloaded this project and the dependencies, reinstalled the NuGet packages and still getting the same issue, however it's now after "Assembling world..." and then VS just crashes.
